diff --git a/plugin/paredit.vim b/plugin/paredit.vim
index 9317ab9..d921333 100644
--- a/plugin/paredit.vim
+++ b/plugin/paredit.vim
@@ -127,8 +127,8 @@ function! PareditInitBuffer()
         nnoremap <buffer> <silent> s            :<C-U>call PareditEraseFwd()<CR>i
         nnoremap <buffer> <silent> D            v$:<C-U>call PareditDelete(visualmode(),1)<CR>
         nnoremap <buffer> <silent> C            v$:<C-U>call PareditChange(visualmode(),1)<CR>
-        nnoremap <buffer> <silent> d            :<C-U>call PareditSetDelete(v:count)<CR>g@
-        vnoremap <buffer> <silent> d            :<C-U>call PareditDelete(visualmode(),1)<CR>
+        " nnoremap <buffer> <silent> d            :<C-U>call PareditSetDelete(v:count)<CR>g@
+        " vnoremap <buffer> <silent> d            :<C-U>call PareditDelete(visualmode(),1)<CR>
         vnoremap <buffer> <silent> x            :<C-U>call PareditDelete(visualmode(),1)<CR>
         vnoremap <buffer> <silent> <Del>        :<C-U>call PareditDelete(visualmode(),1)<CR>
         nnoremap <buffer> <silent> c            :set opfunc=PareditChange<CR>g@
@@ -179,7 +179,10 @@ function! PareditInitBuffer()
             execute 'nnoremap <buffer> <silent> ' . g:paredit_leader.'S  :<C-U>normal! S<CR>'
         else
             " Longer keymaps with <Leader> prefix
-            nnoremap <buffer> <silent> S            V:<C-U>call PareditChange(visualmode(),1)<CR>
+            " nnoremap <buffer> <silent> S            V:<C-U>call PareditChange(visualmode(),1)<CR>
+            nnoremap <buffer> <silent> S            :<C-U>call PareditSplice()<CR>
+            nnoremap <buffer> <silent> <            :<C-U>call PareditMoveLeft()<CR>
+            nnoremap <buffer> <silent> >            :<C-U>call PareditMoveRight()<CR>
             execute 'nnoremap <buffer> <silent> ' . g:paredit_leader.'<  :<C-U>call PareditMoveLeft()<CR>'
             execute 'nnoremap <buffer> <silent> ' . g:paredit_leader.'>  :<C-U>call PareditMoveRight()<CR>'
             execute 'nnoremap <buffer> <silent> ' . g:paredit_leader.'O  :<C-U>call PareditSplit()<CR>'
